### DISCUSSION TITLE
how do you set up a gif command

### DISCUSSION THREAD

**Discussion by anime731** [Rapptz/discord.py#6709] [Q&A]
srry for asking you this but i really need your help for my bot for anyone to use not just me like example in some anime bots if you put !cry there will be an anime character crying ass  the gif thats how the bot will reply.

**Comment by Rapptz** [comment]
This isn't really the place to ask for code. Sorry.

**Comment by zanderp25** [comment]
You can have it send a link to the gif and Discord will automatically replace it with the gif
